### PR TITLE
docs(storage): custom roles

### PR DESCRIPTION
**Pull request by fenos**
## What kind of change does this PR introduce?

docs update

## What is the new behavior?

Document how to use custom roles with Storage

### CODE REVIEW

**Comment by inian** [comment]
LGTM

File: apps/docs/pages/guides/storage/schema/custom-roles.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'storage-custom-roles',
+  title: 'Custom Roles',
```
Comment (charislam) [inline apps/docs/pages/guides/storage/schema/custom-roles.mdx:5]:
```suggestion
  title: 'Custom roles',
```

File: apps/docs/pages/guides/storage/schema/custom-roles.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  description: 'Learn about the storage schema',
+  subtitle: 'Use custom roles with Storage',
+  sidebar_label: 'Schema',
+}
+
+Storage's role-based access control can dynamically adjust to the role specified in the incoming authorization header. By default, it supports:
+
+- **anon**: For unauthenticated access.
+- **authenticated**: For standard user access.
+- **service_role**: For server side access.
```
Comment (charislam) [inline apps/docs/pages/guides/storage/schema/custom-roles.mdx:15]:
```suggestion
- **service_role**: For-server side access.
```

**Review by charislam — APPROVED** [review]

**Comment by joshenlim** [comment]
There already seems to be a custom-roles.mdx [here](https://github.com/supabase/supabase/blob/master/apps/docs/content/guides/storage/schema/custom-roles.mdx#L2) on latest master which conflicts with the content in this PR - will hence be closing this PR but please do feel free to reopen

**Pull request closed without merging**